dd-sdk-android/src/test/kotlin/com/datadog/android/utils/assertj/DeserializedResourceEventAssert.kt

```diff
@@ -0,0 +1,53 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.utils.assertj
+
+import com.datadog.android.rum.model.ResourceEvent
+import org.assertj.core.api.AbstractAssert
+import org.assertj.core.api.Assertions.assertThat
+
+internal class DeserializedResourceEventAssert(actual: ResourceEvent) :
+    AbstractAssert<DeserializedResourceEventAssert, ResourceEvent>(
+        actual,
+        DeserializedResourceEventAssert::class.java
+    ) {
+
+    fun isEqualTo(expected: ResourceEvent): DeserializedResourceEventAssert {
+        assertThat(expected)
+            .usingRecursiveComparison()
+            .ignoringFields("context", "usr", "dd")
```

Comment:
why do we ignore `dd` field here, but not for action, error and view assertions?